single_period.py

```diff
@@ -297,7 +298,7 @@ def solve_cqm(self, max_risk=None, min_return=None, init_holdings=None):
                                                                 label="Example - Portfolio Optimization")
         n_samples = len(self.sample_set['CQM'].record)
 
-        feasible_samples = self.sample_set['CQM'].filter(lambda d: d.is_feasible) 
+        feasible_samples = (self.sample_set['CQM']).filter(lambda d: d.is_feasible) 
```

Comment:
Parentheses not needed here.

Reply:
This looks like the remnants of some debugging that got left in inadvertently. Fixed. Thanks!